python/example/model-downloader/ModelDownloaderExample.ipynb

```diff
@@ -198,9 +198,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from sparknlp.pretrained.pipeline.en import SentimentPipeline\n",
-    "\n",
-    "SentimentPipeline.annotate(\"This is a good movie!!!\")"
+    "PretrainedPipeline(\"pipeline_vivekn\").annotate(\"This is a good movie!!!\")"
```

Comment:
Shouldn't this be `sentiment_vivken`? We know already it is a pipeline due to `PretrainedPipeline`, I think `vivekn` on its own is not clear, so we can combine the annotator with its algorithm/method: `sentiment_vivekn`.

Reply:
disregard model and pipeline names for now :) they should be all remade from scratch